tests/test_config_reader.py

```diff
@@ -387,6 +395,42 @@ def test_construct_stacks_with_valid_config(
         assert {str(stack) for stack in all_stacks} == expected_stacks
         assert {str(stack) for stack in command_stacks} == expected_command_stacks
 
+    def test_construct_stacks_with_disable_rollback_command_param(self):
+        project_path, config_dir = self.create_project()
+
+        rel_path = "A/1.yaml"
+        config = {
+            "region": "region",
+            "project_code": "project_code",
+            "template_path": rel_path,
+        }
+
+        abs_path = os.path.join(config_dir, rel_path)
+        self.write_config(abs_path, config)
+        self.context.project_path = project_path
+        self.context.command_params["disable_rollback"] = True
+        config_reader = ConfigReader(self.context)
+        all_stacks, command_stacks = config_reader.construct_stacks()
+        assert "disable_rollback=True" in str(all_stacks)
```

Comment:
This strikes me as a bit odd. Why are you asserting off the string of the stacks? Why not loop the stacks and check that they all have `disable_rollback` equal to True? Technically this test is only testing that just one Stack in the list of stacks has `disable_rollback`.

Reply:
this test doesn't require testing with multiple stacks.  It's is deliberately testing with just one stack therefore no need to loop?

Reply:
It still isn't really proper to test off the `str` output of a thing unless your test is on the `__str__` method itself. Rather, you should assert off the property you're _actually_ testing for. Like this:

```suggestion
        assert list(all_stacks)[0].disable_rollback
```

Reply:
updated the assert